packages/mco/components/mco-dashboard/data-policy/summary-card/summary-card.tsx

```diff
@@ -45,12 +45,12 @@ const getClusterSummary = (
     drClusters?.forEach((cluster) => {
       summaryMap.applications.totalCount =
         summaryMap.applications.totalCount +
-        drClusterAppsMap[cluster].totalAppSetsCount;
+        drClusterAppsMap[cluster].totalAppCount;
       summaryMap.applications.protectedCount =
         summaryMap.applications.protectedCount +
-        drClusterAppsMap[cluster].protectedAppSets.length;
+        drClusterAppsMap[cluster].protectedApps.length;
       const isClusterHealthy = ValidateManagedClusterCondition(
-        drClusterAppsMap[cluster]?.managedCluster,
+        drClusterAppsMap[cluster]?.managedCluster[0],
```

Comment:
```suggestion
        drClusterAppsMap[cluster]?.managedCluster,
```
managedCluster is not list type